wordpress-mobile/WordPress-Android
# Stats revamp v2: Fix total card problems

This fixes problems on total cards:
- Adds view more button to Followers card on Total Followers detail screen (This button is on current iOS version and in Figma design),
- Removes the date picker from Total Followers and Total Comments cards (my commit caf9893d11720fc55187d8fc320d68b54c47d0cc message is wrong)
- Adds date picker functionality to Total Likes card on the detail screen.
- Renames Publicize to Social Media Connections.

To test:

Setup:
1. Go to App Settings on Jetpack app (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
2. Select Debug Settings
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app
4. Launch / Re-Launch app
5. Go to Stats either using quick links or menu
6. Switch to Insights tab if necessary
7. Make sure Total Likes, Total Followers and Total Comments cards are visible on Insights tab. If not present then add ithem through stats management using either [+ add new stats card ] at the bottom or clicking ⚙️ icon at the top right hand corner.

Test:

- Click on View More on Total Followers card.
- Ensure there is view more button bottom of Followers card. (It only appears if there are more than 6 followers)
- Navigate back and tap VIEW MORE on Total Followers card.
- Ensure there is no date picker on top of the screen.
- Navigate back and tap VIEW MORE on Total Comments card.
- Ensure there is no date picker on top of the screen. 

## Regression Notes
1. Potential unintended areas of impact
Total cards blocks, and their detail screens.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually.

3. What automated tests I added (or what prevented me from doing so)
Updated current unit tests.

PR submission checklist:
